README.md

```diff
@@ -139,8 +139,9 @@ const OrbitDB = require('orbit-db')
 
 const ipfs = IpfsApi('localhost', '5001')
 const orbitdb = new OrbitDB(ipfs)
-const db = await orbitdb.log('hello')
-...
+const db = orbitdb.log('hello').then(db => {
```

Comment:
The call will still return a promise so I think we should get rid of the `const db = `

Reply:
Removed!